src/views/templates/actions/hooks/useVirtualMachineTemplatesActions.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+
+import { TemplateModel, V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import DeleteModal from '@kubevirt-utils/components/DeleteModal/DeleteModal';
+import { useModal } from '@kubevirt-utils/components/ModalProvider/ModalProvider';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Action, k8sDelete } from '@openshift-console/dynamic-plugin-sdk';
+
+import { getTemplateProviderName } from '../../utils/selectors';
+
+type useVirtualMachineTemplatesActionsProps = (template: V1Template) => Action[];
+
+const useVirtualMachineTemplatesActions: useVirtualMachineTemplatesActionsProps = (
+  template: V1Template,
+) => {
+  const { t } = useKubevirtTranslation();
+  const isRedHatTemplate = getTemplateProviderName(template) === 'Red Hat';
```

Comment:
we need to disable the delete option for all common templates,
this check will not catch all common templates since in some cases the annotation `openshift.io/provider-display-name` can hold a value `KubeVirt` and the delete will be enabled.
I suggest to change this check to something a bit more accurate,
maybe checking if the label `app.kubernetes.io/name: common-templates` exists, @yaacov wdyt?

Reply:
> disable the delete option for all common templates

We should not create/edit/delete templates that are controlled and deleted by SSP operator,
so when disabling actions on this template we need to test for templates controlled by SSP operator.
The label to use is `template.kubevirt.io/type: base` [1], and the name we call this templates is "common templates"
( maybe we should rename them "base templates" instead ? )

[1]https://github.com/kubevirt/ssp-operator/blob/master/internal/operands/common-templates/constants.go#L6
    https://github.com/kubevirt/ssp-operator/blob/master/internal/operands/common-templates/constants.go#L5

common template is defined by:
`const isCommonTemplate = metadate.labels['template.kubevirt.io/type'] === "base"`
so if we want to know if it's common template this is the test

provided by RedHat is defined by:
`const isProvidedByRedHat = metadata.annotations['template.kubevirt.io/provider'] === 'Red Hat'`
but the provider is not checked by SSP operator, and it is not indicative for actions being available or not on the back end.